client-registry-jempi/package-metadata.json

```diff
@@ -41,7 +35,6 @@
     "JEMPI_API_MEMORY_LIMIT": "3G",
     "JEMPI_API_MEMORY_RESERVE": "500M",
     "JEMPI_KAFKA_TOPICS": "JeMPI-async-preprocessor,JeMPI-patient-controller,JeMPI-patient-em,JeMPI-patient-linker,JeMPI-mu-linker,JeMPI-notifications",
```

Comment:
I think this is something we need to care about, maybe we can put all these topics along side with the other topics in kafka folder, and they can be overwritten according to each implementation ?